src/main/java/GeneralService/OnlineServiceHandler.java

```diff
@@ -0,0 +1,13 @@
+package GeneralService;
+
+import onlineservices.Service;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.List;
+
+@org.springframework.stereotype.Service
```

Comment:
The Service class from OnlineServices was changed to OnlineService. update this annotation to @Service